### DISCUSSION TITLE
how to specify multiple c_arg

### DISCUSSION THREAD

**Discussion by jianhe-fun** [mesonbuild/meson#12619] [Q&A] [answered]
context: https://wiki.postgresql.org/wiki/Meson 
```
meson setup ${BUILD} \
        -Dprefix=${PG_PREFIX} \
        -Dpgport=5462 \
        -Dcassert=true \
        -Db_coverage=true \
        -Dicu=enabled \
        -Dbuildtype=debug \
        -Dwerror=true \
        -Dc_args='-Wunused-variable' \
        -Dc_args='-Wuninitialized' \
        -Dc_args='-Wreturn-type'

```
will only give me
c_args                 : -Wreturn-type             

how can I specify '-Wreturn-type' and '-Wuninitialized' and '-Wunused-variable'

**Comment by dcbaker** [comment, marked as answer]
Usually it's sufficient to just do `-Dc_args='-Wunused-variable -Wunintialized -Wreturn-type'` and meson will split that on the whitespace, if you get into situations where you need to escape white space in arguments you can pass a quoted meson array: `-Dc_args="['-A with spaces', '-Wunused-variable']"`
